.travis.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - stage: Comprehensive tests
       python: 3.6
     - python: 3.7
-#    - python: 3.9-dev
+    - python: 3.9-dev
 
     - python: 3.6
       env: USE_DEBUG=1
```

Comment:
at some point we should upgrade the debug run to 3.7 or even 3.8

Reply:
> at some point we should upgrade the debug run to 3.7

We are waiting on Ubuntu Focal Fossa (20.04 LTS) which has 3.8 as default. Looks like that may now be available.